docs/api_v0.yml

```diff
@@ -1104,6 +1104,48 @@ components:
           type: string
           description: Profile image (90x90)
 
+    Organization:
+      description: Organization
+      type: object
+      properties:
+        type_of:
+          type: string
+        username:
+          type: string
+        name:
+          type: string
+        summary:
+          type: string
+          nullable: true
+        twitter_username:
+          type: string
+          nullable: true
+        github_username:
+          type: string
+          nullable: true
+        url:
+          type: string
+          format: url
+        location:
+          type: string
+          nullable: true
+        tech_stack:
+          type: string
+          nullable: true
+        tag_line:
+          type: string
+          nullable: true
+        story:
+          type: string
+          nullable: true
+        joined_at:
+          description: Date of joining (formatted with strftime `"%b %e, %Y"`)
```

Comment:
I think we can remove the part in the parenthesis here and then merge :D , also can you add `format: date-time` to the spec of this field ?

Reply:
No problem. I just pushed up the changes for that. Thanks